src/components/PostResults.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import PostCard from './PostCard';
+import postData from './dummyData';
+import { Grid } from '@material-ui/core';
+
+export default function PostResults() {
+  const [posts, setPost] = useState([]);
+  useEffect(() => {
+    setPost(postData);
+  }, []);
+
+  return (
+    <React.Fragment>
+      {posts.map((post) => (
+        <Grid item xs={12}>
```

Comment:
```suggestion
        <Grid item xs={12} className={classes.root}>
```

Reply:
This isn't a root el so shouldn't have a root class